sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/networking.py

```diff
@@ -207,3 +210,23 @@ def _from_rest_object(cls, obj: RestManagedNetwork) -> "ManagedNetwork":
             network_id=obj.network_id,
             status=obj.status,
         )
+
+
+class ManagedNetworkProvisionStatus:
+    """ManagedNetworkProvisionStatus."""
+
```

Comment:
make sure to add some small docstring so that customers can see it in the reference documentation